README.md

```diff
@@ -285,3 +286,189 @@ bundle exec cap prod deploy # for the prod servers
 
 ### Resque Pool
 The Resque Pool admin interface is available at `<hostname>/resque/overview`.
+
+## API
+### `GET /objects/:druid`
+Return the PreservedObject model for the object.
+
```

Comment:
Not blocking but I thinking in addition to the `curl` examples, would adding an example of how the API is used with the `preservation-catalog` gem in other Ruby projects would be helpful.